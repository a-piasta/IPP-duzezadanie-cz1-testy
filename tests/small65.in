getRouteDescription;819
addRoad;rsjztdzeddntunhrwrdgumgsly;hcqrkgcyqlovfhwnbtxosnubtazqz;6;-3
removeRoad;hcqrkgcyqlovfhwnbtxosnubtazqz;rsjztdzeddntunhrwrdgumgsly
getRouteDescription;970
extendRoute;50;sevgfuptaodztyvmajqysarv
addRoad;sevgfuptaodztyvmajqysarv;drrlimxifaj;3;4
addRoad;drrlimxifaj;hiravbosjv;7;-12
addRoad;sjoefjylgfqaujckpedpzr;sjoefjylgfqaujckpedpzr;24;-5
newRoute;574;vyaricxnxuztcgthpapnswoiie;vyaricxnxuztcgthpapnswoiie
newRoute;478;dmywgogzkxetutzoe;rsjztdzeddntunhrwrdgumgsly
getRouteDescription;574
newRoute;208;hiravbosjv;dmywgogzkxetutzoe
extendRoute;478;rsjztdzeddntunhrwrdgumgsly
repairRoad;bgyrzjycjqpamwjkmkdjyxxszssy;bgyrzjycjqpamwjkmkdjyxxszssy;8
newRoute;279;hcqrkgcyqlovfhwnbtxosnubtazqz;hcqrkgcyqlovfhwnbtxosnubtazqz
repairRoad;gojokjodrsagsvilzqehfgyvyu;sevgfuptaodztyvmajqysarv;6
extendRoute;478;vyaricxnxuztcgthpapnswoiie
getRouteDescription;478
addRoad;rsjztdzeddntunhrwrdgumgsly;hiravbosjv;14;-4
newRoute;383;vyaricxnxuztcgthpapnswoiie;vyaricxnxuztcgthpapnswoiie
addRoad;sjoefjylgfqaujckpedpzr;sevgfuptaodztyvmajqysarv;24;-9
extendRoute;574;sjoefjylgfqaujckpedpzr
repairRoad;vyaricxnxuztcgthpapnswoiie;hiravbosjv;11
repairRoad;sjoefjylgfqaujckpedpzr;dmywgogzkxetutzoe;2
getRouteDescription;279
extendRoute;208;vyaricxnxuztcgthpapnswoiie
extendRoute;50;hcqrkgcyqlovfhwnbtxosnubtazqz
getRouteDescription;208
getRouteDescription;478
extendRoute;279;drrlimxifaj
removeRoad;drrlimxifaj;sevgfuptaodztyvmajqysarv
extendRoute;383;hcqrkgcyqlovfhwnbtxosnubtazqz
repairRoad;gojokjodrsagsvilzqehfgyvyu;bgyrzjycjqpamwjkmkdjyxxszssy;7
newRoute;688;rsjztdzeddntunhrwrdgumgsly;hiravbosjv
extendRoute;383;drrlimxifaj
newRoute;609;drrlimxifaj;sevgfuptaodztyvmajqysarv
addRoad;hiravbosjv;drrlimxifaj;11;-9
newRoute;18;hiravbosjv;hcqrkgcyqlovfhwnbtxosnubtazqz
addRoad;hiravbosjv;vyaricxnxuztcgthpapnswoiie;24;11
addRoad;sjoefjylgfqaujckpedpzr;drrlimxifaj;7;-7
newRoute;10;rsjztdzeddntunhrwrdgumgsly;drrlimxifaj
repairRoad;vyaricxnxuztcgthpapnswoiie;drrlimxifaj;-4
newRoute;90;bgyrzjycjqpamwjkmkdjyxxszssy;bgyrzjycjqpamwjkmkdjyxxszssy
getRouteDescription;95
removeRoad;sevgfuptaodztyvmajqysarv;sjoefjylgfqaujckpedpzr
getRouteDescription;688
addRoad;sevgfuptaodztyvmajqysarv;drrlimxifaj;18;-15
newRoute;886;vyaricxnxuztcgthpapnswoiie;hiravbosjv
addRoad;sjoefjylgfqaujckpedpzr;drrlimxifaj;1;8
newRoute;347;bgyrzjycjqpamwjkmkdjyxxszssy;hiravbosjv
repairRoad;vyaricxnxuztcgthpapnswoiie;gojokjodrsagsvilzqehfgyvyu;8
removeRoad;drrlimxifaj;sevgfuptaodztyvmajqysarv
repairRoad;rsjztdzeddntunhrwrdgumgsly;sjoefjylgfqaujckpedpzr;-13
getRouteDescription;18
repairRoad;sjoefjylgfqaujckpedpzr;dmywgogzkxetutzoe;0
extendRoute;10;dmywgogzkxetutzoe
repairRoad;sjoefjylgfqaujckpedpzr;rsjztdzeddntunhrwrdgumgsly;-8
getRouteDescription;304
