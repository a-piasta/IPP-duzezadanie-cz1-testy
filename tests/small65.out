1: 
2: TAK
3: TAK
4: 
5: NIE
6: TAK
7: TAK
8: NIE
9: NIE
10: NIE
11: 
12: NIE
13: NIE
14: NIE
15: NIE
16: NIE
17: NIE
18: 
19: TAK
20: NIE
21: TAK
22: NIE
23: NIE
24: NIE
25: 
26: NIE
27: NIE
28: 
29: 
30: NIE
31: TAK
32: NIE
33: NIE
34: TAK
35: NIE
36: NIE
37: NIE
38: NIE
39: TAK
40: TAK
41: TAK
42: NIE
43: NIE
44: 
45: TAK
46: 688;rsjztdzeddntunhrwrdgumgsly;14;-4;hiravbosjv
47: TAK
48: TAK
49: NIE
50: NIE
51: NIE
52: TAK
53: NIE
54: 
55: NIE
56: NIE
57: NIE
58: 
